packages/playback-core/src/request-errors.ts

```diff
@@ -0,0 +1,255 @@
+import type { LoaderResponse } from 'hls.js';
```

Comment:
NOTE: We may want to clean up our organization of error modules. I'm definitely amenable to relocating this, consolidating, etc.